_posts/2017-05-07-Blockchain-Introduction.md

```diff
@@ -0,0 +1,422 @@
+People say blockchain is as important as the introduction of the internet. 
+The internet is a worldwide network to __share information__ with one another, but it is far less suitable for transferring value.
+If you send someone a file, it is always a copy of your file, which means you and the receiver are both in possession of the file.
+As we already stated, that is ideal for sharing information, but not applicable for money, certificate of ownership, and so forth.
+And the latter is exactly what blockchain enables: digitalizing and transferring such values.
+
+
+Let's take a look at the underlying decentralized ledger technology.
+We believe blockchain's definition is a good starting point:
+
+> "Blockchain is a type of __distributed database__ that stores a __permanent__ and __tamper-proof__ ledger of __transaction data__."
+
+
+### TL;DR version
+
+Blockchain is a __decentralized immutable data structure__.
+In short the blockchain is a network of computers, called nodes. 
+Every node has the same copy of the database which they manage together. 
+A transaction is encrypted and signed on a mathematical way. 
+When a transaction is saved in the blockchain, it is duplicated across all nodes in the network.
+That's why we talk of blockchain as __distributed ledger technology__, a ledger of transactions, distributed across a computer network.
+Transactions are bundled in one __block__ before they are validated by other nodes.
+Once the network reached consensus about the validity of these transactions, the block is appended to the existing __chain__ of blocks.
+The block stores the validated transactions together with a hash and a reference to the previous block.
+Stored transactions cannot be undone, as this would invalidate all hashes in the chain.
+
+
+### Now a little more in detail...
+
+Transactions are broadcasted to the network for miners to __mine__. 
+They assess the non-validated transactions on the memory pool by solving a mathematical puzzle. 
+A miner builds a block containing all transactions, a proof of work that the puzzle was solved (also known as the __block root hash__, which is also the ID of the block) and a hash to the previous block.
+
+A block also contains the following items:
+
+* A timestamp
+* a nonce
+* and a __merkle root hash__
+
+A __merkle root__ does not verify transactions, but verifies a set of transactions.
+Transaction IDs are hashes of the transaction, and the merkle tree is constructed from these hashes.
+It means that if a single detail in any of the transactions changes, so does the merkle root. 
+It also means that if the exact same transactions are listed in a different order, the merkle root will also change.
+So the merkle root is cryptographic proof of the transactions in the block together with the order they are in.
+
+The __nonce number__ specifies that the block root hash must start with a specified number, like four leading zeros.
+This way**,** blockchain is very secure. 
```

Comment:
I think the asterixes were copy-pasted?
I only included them in the review to mark the comma in bold.